README.md

```diff
@@ -1 +1,46 @@
-# leantime-defaulttickettemplate
+# Leantime Ticket Template
+
+Allows administrators to create and configure a ticket templates per project.
+
+## Installation
+
+Clone this repository into your Leantime plugin folder:
+
+``` shell
+git clone https://github.com/ITK-Leantime/leantime-tickettemplate.git app/Plugins/TicketTemplate
+```
+
+Install and enable the plugin:
+
+``` shell
+bin/leantime plugin:install leantime/ticket-template
+bin/leantime plugin:enable leantime/ticket-template
+```
+
+Alternatively, navigate to `/plugins/myapps` and activate the
+`leantime/ticket-template` plugin.
+
+## Usage
+
+Go to plugin settings (`/TicketTemplate/settings`),
+which requires at least administrator (40) role,
+to configure ticket templates.
+
+**Note**: Currently, translations are read before the plugin register is handled,
+resulting in plugin translations not being considered.
+Go to Settings (`/users/editOwn#settings`) and save to refresh
+plugin translations.
```

Comment:
Would it be possible to fix this in the enable plugin commands? Known that would be a PR to core and may be out-of-scope!